docs/core-concepts.md

```diff
@@ -104,6 +123,7 @@ await frame.fill('#username-input', 'John');
 
 - [class `Page`](./api.md#class-page)
 - [class `Frame`](./api.md#class-frame)
+- [Navigation and loading](loading.md)
```

Comment:
It is very easy for me to assume this is a link to the api.md and never click it. Do "To learn more about page navigation and loading, read this document".